CONTRIBUTING.md

```diff
@@ -48,6 +48,22 @@ Examples:
   - If the PR has `BREAKING CHANGE`: in its description is a breaking change
 - `remove!: description of the PR`: The commit removes a feature from the product. Typically features are deprecated first for a period of time before being removed. Removing a feature is a breaking change (correlating with MAJOR in Semantic Versioning).
 
+## Discovering New API features
```

Comment:
Can we additionally add the steps for updating a major version? https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md#sdk-major-release-update-procedure

Reply:
updated